<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wUser</class>
 <widget class="QWidget" name="wUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="usuarioLabel">
       <property name="text">
        <string>Usuario:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="usuarioLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="contraseALabel">
       <property name="text">
        <string>Contraseña: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="contraseALineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="contraseACortaLabel">
       <property name="text">
        <string>Contraseña Corta:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="contraseACortaLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="nombreLabel">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nombreLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="apellidoLabel">
       <property name="text">
        <string>Apellido</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="apellidoLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="telefonoLabel">
       <property name="text">
        <string>Telefono:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="telefonoLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="fechaNacimienoLabel">
       <property name="text">
        <string>Fecha Nacimieno:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDateEdit" name="fechaNacimienoDateEdit">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1980</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="activoLabel">
       <property name="text">
        <string>Activo</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="activoCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="adminLabel">
       <property name="text">
        <string>Admin</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="adminCheckBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
